django_socio_grpc/proto_serializers.py

```diff
@@ -27,14 +32,57 @@ def __init__(self, *args, **kwargs):
         message = kwargs.pop("message", None)
         self.stream = kwargs.pop("stream", None)
         self.message_list_attr = kwargs.pop(LIST_ATTR_MESSAGE_NAME, DEFAULT_LIST_FIELD_NAME)
+        # INFO - AM - 04/01/2023 - Need to manually define partial before the super().__init__ as it's used in populate_dict_with_none_if_not_required that is used in message_to_data that is call before the super init
+        self.partial = kwargs.get("partial", False)
         if message is not None:
             self.initial_message = message
             kwargs["data"] = self.message_to_data(message)
         super().__init__(*args, **kwargs)
 
     def message_to_data(self, message):
         """Protobuf message -> Dict of python primitive datatypes."""
-        return message_to_dict(message)
+        data_dict = message_to_dict(message)
+        data_dict = self.populate_dict_with_none_if_not_required(data_dict, message)
+        return data_dict
+
+    def populate_dict_with_none_if_not_required(self, data_dict, message=None):
```

Comment:
The PARTIAL_UPDATE_FIELD_NAME should already be in data_dict
```suggestion
    def populate_dict_with_none_if_not_required(self, data_dict):
```